deploy/kubernetes/helm-charts/altinn-runtime/values.yaml

```diff
@@ -155,8 +156,14 @@ volumes:
     secret:
       secretName: altinn-appsettings-secret
 
-service:
-  ports:
-    - port: 5005
-      targetPort: 5005
-      protocol: TCP
+ingress:
+  - metadata:
+      name: altinn-runtime-base-path
+      annotations:
+        traefik.ingress.kubernetes.io/request-modifier: "ReplacePathRegex: ^/.*/.*/staticfiles(.*)$ $1"
+        traefik.frontend.priority: "100"
+        # kubernetes.io/ingress.class: nginx
+        # kubernetes.io/tls-acme: "true"
```

Comment:
The "commented out"-code should be kept for future reference or removed?

Reply:
Removed.